st/clitests/awss3api_spec.py

```diff
@@ -94,6 +94,49 @@ def delete_object_list_file(file_name):
 #******** Create Bucket ********
 AwsTest('Aws can create bucket').create_bucket("seagatebucket").execute_test().command_is_successful()
 
+#******** Verify pagination and NextToken in List objects V1  **********
+# Step 1. Create total 110 objets in bucket
+obj_110_list = []
+for x in range(110):
+    key = "object%d" % (x)
+    AwsTest(('Aws Upload object: %s' % key)).put_object("seagatebucket", "3Kfile", 3000, key_name=key)\
+        .execute_test().command_is_successful()
+    obj_110_list.append(key)
+
+# Step 2.
+# Create object list file
+object_list_file = create_object_list_file("obj_list_110_keys.json", obj_110_list, "true")
+
+# Step 3. List first 100 objects. This will result into NextToken != None
+# NextToken will point to object9 - the last object key in the response.
```

Comment:
minor correction: NextToken will point to object99 !!

Reply:
NextToken will point to the last key in the result set. The last key in this context will be 'object9' (and not 'object 99').